<launch>
  <!-- Launch the tracking node -->
  <node pkg="zbar_ros" type="barcode_detector" name="barcode_detector" output="screen">
    <param name = "image_topic" type="string" value = "/cameras/head_camera/image" />
    <param name = "show_cv_window" type="bool" value = "true" />
  </node>
</launch>
